scripts/contract.mjs

```diff
@@ -23,8 +23,6 @@ export async function deploy(contract, args = undefined) {
       process.env.PRIVATE_KEY,
       "--rpc-url",
       process.env.ETH_RPC_URL,
-      "--chain",
```

Comment:
what does this even do? is it for etherscan?

Reply:
great question & I'm not sure.

forge docs are pretty poopy.

Reply:
All I know is it works w/o it and with it I get errors when I try to deploy onto `mumbai`.

Reply:
@SweetmanTech Foundry has updated since these scripts were written.